code/modules/flufftext/Dreaming.dm

```diff
@@ -1,20 +1,19 @@
 
 var/list/dreams = list(
 	"an ID card","a bottle","a familiar face","a crewmember","a toolbox","a security officer","the captain",
-	"voices from all around","deep space","a doctor","the engine","a traitor","an ally","darkness",
+	"voices from all around","deep space","a doctor","the engine","a unitologist","an ally","darkness",
 	"light","a scientist","a monkey","a catastrophe","a loved one","a gun","warmth","freezing","the sun",
 	"a hat","a ruined station","a planet",MATERIAL_PHORON,"air","the medical bay","the bridge","blinking lights",
-	"a blue light","an abandoned laboratory","NanoTrasen", "pirates", "mercenaries","blood","healing","power","respect",
+	"a blue light","an abandoned laboratory","EarthGov","CEC","pirates", "mercenaries","blood","healing","power","respect",
 	"riches","space","a crash","happiness","pride","a fall","water","flames","ice","melons","flying","the eggs","money",
-	"the chief engineer","the research director","the chief medical officer",
```

Comment:
not sure why it says I deleted the whole section

Reply:
You could probably trim more of the dreams for jobs we don't have anymore.

Stuff like, Medical Contractor, Roboticist, etc.

Reply:
done, anything i missed?

Reply:
The CEC pilot, the passenger, the master at arms, the prospector, and the stowaway are the only ones left I'd say could go.

Reply:
all done